src/Generation/FieldDetails.php

```diff
@@ -260,9 +275,22 @@ public function getOneofDesc()
         return !$this->isOneOf ? null : $this->containingMessage->getOneofDecl()[$this->oneOfIndex];
     }
 
-    public function exampleValue(SourceFileContext $ctx)
-    {
-        if ($this->desc->desc->isRepeated()) {
+    /**
+     * Create an example value based off the type of this field.
+     *
+     * @param SourceFileContext $ctx The context this field is attached to.
+     * @param bool $formatStringWithBrackets Whether or not to format a string
+     *             value as "['VALUE']" or "'value'".
```

Comment:
nit, but I believe this should be
```suggestion
     *             value as "'[VALUE]'" or "'value'".
```